src/models/streaming/StreamingResponses.ts

```diff
@@ -11,10 +11,9 @@ export interface StreamingResponse {
  * Represents a hypothesis response from the rev.ai streaming service. Both final and partial.
  */
 export interface StreamingHypothesis extends StreamingResponse {
-    transcript?: string;
     ts?: number;
     end_ts?: number;
-    elements: Element[];
+    elements?: Element[];
```

Comment:
I'm not sure this `?` is needed since `elements` can just be an empty array if it doesn't exist. Let's check once punctuation is on production